src/3rdparty/kissfft/tools/Makefile

```
@@ -1,62 +1,101 @@
+DUMPHDR = dumphdr-$(KISSFFT_DATATYPE)-openmp
 endif
 
-ifneq ("$(KISS_FFT_USE_ALLOCA)","")
-	CFLAGS+= -DKISS_FFT_USE_ALLOCA=1
-endif 
-CFLAGS+= $(CFLAGADD)
+#
+# Target: "make all"
+#
 
+all: $(FFTUTIL) $(FASTFILT) $(FASTFILTREAL) $(PSDPNG)
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)